Frameworks/UIKit/UITableViewCell.mm

```diff
@@ -1191,7 +1191,10 @@ - (void)layoutSubviews {
     if (accessoryView) {
         CGRect accessoryRect = { 0 };
 
-        if ([accessoryView isKindOfClass:[UITextField class]]) {
+        // TODO: on reference platform, when UILabel is the accessory view, it uses its own intrinContentsize (similar to UITextField)
```

Comment:
Seems like fixing bugs here would be preferable at this point.

---
In reply to: [97385306](https://github.com/Microsoft/WinObjC/pull/1768#discussion_r97385306) [](ancestors = 97385306)